src/ui/shared/channels.mock.ts

```diff
@@ -149,6 +149,8 @@ class WalletPortMock {
       this.setPreference({
         preferences: args[0] as Partial<WalletRecord['publicPreferences']>,
       });
+    } else if (method === 'getWalletGroupByAddress') {
+      return mockRecord.walletManager.groups[0];
```

Comment:
maybe actually do a search here?

Reply:
May be... I tried to fix this quickly. Will add a TODO for the future :)